wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/DeviceDetailsView.swift

```diff
@@ -127,6 +121,18 @@ struct DeviceDetailsView: View {
         .onDisappear {
             dismissedView?()
         }
+        .sheet(isPresented: $isCertificateViewPresented) {
+            if let certificate = viewModel.e2eIdentityCertificate {
+                E2EIdentityCertificateDetailsView(
+                    certificateDetails: certificate.certificateDetails.uppercased(),
```

Comment:
Instead of `uppercased()` can we replace it with `.localizeUppercase` so we can make sure the end result is localized?